src/gtimelog/tests/test_timelog.py

```diff
@@ -11,18 +11,10 @@
 import time
 import unittest
 import sys
```

Comment:
The flake8 job failed with

> src/gtimelog/timelog.py:13:1: F401 'sys' imported but unused

so this line needs to go away too.

Reply:
Hmm, apparently not quite.

Reply:
Doctests use it, so it’s better to keep it. However, `# noqa: F401` doesn’t seem to work :disappointed:

Reply:
Mea culpa, I attached this comment on the wrong file!

It's not src/gtimelog/tests/test_timelog.py that needs to drop the `import sys`, but src/gtimelog/timelog.py.